src/dom-helpers.js

```diff
@@ -32,17 +28,17 @@ export function findHighest(current, componentNode, ignoreClass) {
   // thinking in terms of Dom node nesting, running counter
   // to React's 'you shouldn't care about the DOM' philosophy.
   while (current.parentNode) {
-    if (isNodeFound(current, componentNode, ignoreClass)) {
-      return true;
+    if (current === componentNode || hasClass(current, ignoreClass)) {
+      return false;
     }
     current = current.parentNode;
   }
-  return current;
+  return true;
 }
 
 /**
  * Check if the browser scrollbar was clicked
  */
-export function clickedScrollbar(evt) {
+export function clickedBrowserScrollbar(evt) {
   return document.documentElement.clientWidth <= evt.clientX || document.documentElement.clientHeight <= evt.clientY;
```

Comment:
Why limit it only to the browser's scrollbar? please review my PR about it :-)

Reply:
@nivos-saar This PR is about slimming down the core API. Once it gets merged in (hopefully) I want to revisit your PR which will likely need to be adjusted a little. I didn't want to include too much in this PR - that's why I have not included your changes. This one is focused on refactor only.